### PR TITLE
Updated Trusted Launch artifacts

### PR DESCRIPTION
**Pull request by prasadmsft**
Updated Trusted Launch artifacts

# PR Checklist

Check these items before submitting a PR... 

[Contribution Guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/README.md)

[Best Practice Guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/best-practices.md)

- [x] - Please check this box once you've submitted the PR if you've read through the Contribution Guide and best practices checklist.

## Changelog

* Added Trusted launch artifacts: security type, image data, uefi settings, attestation extension relevant vars, resource.
*
*

### CODE REVIEW

**Comment by azure-quickstarts** [comment]
@cynthn - check this PR for updates that may be needed to documentation that references this sample.  [This is an automated message. You are receiving it because you are listed as the docOwner in metadata.json.]

**Comment by azure-quickstarts** [comment]
@cynthn - check this PR for updates that may be needed to documentation that references this sample.  [This is an automated message. You are receiving it because you are listed as the docOwner in metadata.json.]

**Comment by prasadmsft** [comment]
@sirfamelin - Please review

**Comment by azure-quickstarts** [comment]
@cynthn - check this PR for updates that may be needed to documentation that references this sample.  [This is an automated message. You are receiving it because you are listed as the docOwner in metadata.json.]

File: quickstarts/microsoft.compute/vm-simple-linux/main.bicep [hunk 1]
```diff
@@ -58,8 +84,20 @@ var linuxConfiguration = {
...
+    secureBootEnabled: true
+    vTpmEnabled: true
+  }
+  securityType: securityType
+}
+var extensionName = 'GuestAttestation'
+var extensionPublisher = 'Microsoft.Azure.Security.LinuxAttestation'
+var extensionVersion = '1.0'
+var maaTenantName = 'GuestAttestation'
+var maaEndpoint = substring('emptystring', 0, 0)
```
Comment (sirfamelin) [inline quickstarts/microsoft.compute/vm-simple-linux/main.bicep:98]: Why not just hard code these values in the extension config?

**Pull request merged**